dist/featureFlags.js

```diff
@@ -170,18 +185,29 @@ function FeatureFlags($q, featureFlagOverrides, initialFlags) {
             return angular.isArray(newFlags) ? resolve(updateFlagsAndGetAll(newFlags)) : updateFlagsWithPromise(newFlags);
         },
 
+        setEnvironment = function(value) {
+            envir = value;
+            featureFlagOverrides.setEnvironment(value);
+        },
+
+        changeEnvironmentVal = function(flag, value) {
+            Object.keys(flag.environments).forEach(function(env) {
+                flag.environments[env] = value;
+            });
+        },
+
         enable = function(flag) {
-            flag.active = true;
+            changeEnvironmentVal(flag, true);
```

Comment:
This doesn't look like it has the right changes in it. I think you'll need to do a dist / release again and check that in? It's weird that he checks in compiled files anyway...